src/ProgressBar.tsx

```diff
@@ -1,13 +1,15 @@
-import React from 'react'
+// @ts-ignore @styled-system/prop-types does not provide type definitions
```

Comment:
What do you think about going ahead and removing prop-types here, rather than ignoring the missing types?

Reply:
Dropping propTypes break the tests and I'd prefer to update the tests in a separate PR.